src/style.css

```diff
@@ -1 +1,82 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+:root {
+  --focus_color: #00acdc;
+}
+
+html {
+  font-family: Roboto , sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+body {
+  margin: 0;
+}
+
+.main_header {
+  display: flex;
+  width: 100%;
```

Comment:
```suggestion
```
There's no need to do it as `display: flex;` makes the element 100% in width anyway.

Reply:
`width:100%` removed